apps/ui/src/components/Widget/Futarchy.vue

```diff
@@ -0,0 +1,106 @@
+<script setup lang="ts">
+import { z } from 'zod';
+import { _n, getUrl } from '@/helpers/utils';
+import { Proposal } from '@/types';
+
+const props = defineProps<{ proposal: Proposal }>();
+
+const FUTARCHY_API_URL =
+  import.meta.env.VITE_FUTARCHY_API_URL ?? 'https://stag.api.tickspread.com';
+const FUTARCHY_LOGO_URL =
+  'ipfs://bafkreigougs774ow3qwkb3kc5ftkpz43cfueputpmkii2l5meuaeivkiqq';
```

Comment:
Could we embed it as SVG icon or at least have it hosted locally without IPFS references for core UI images?